docs/config.rst

```diff
@@ -1,11 +1,606 @@
 
-Local overrides
----------------
+The project ``cumulusci.yml`` file is located in the ``project`` diretory. You can override aspects of the universal ``cumulusci.yml`` file here by defining project dependencies, configuring new tasks and flows, customizing standard tasks and flows to meet your project's needs, and so much more! 
+
+    .. Important:: When ``cumulusci.yml`` is referenced in this documentation, it refers to the project ``cumulusci.yml`` file unless otherwise specified.
+
+
+
+``cumulusci.yml`` Structure
+---------------------------
+
+There are six top-level sections of a ``cumulusci.yml`` file.
+
+* ``project``: This section is largely generated by running ``cci project init``. It contains information pertaining to the project's package information (if any) and GitHub information.
+
+* ``tasks``: This section defines the tasks that are available to run in your project. See `task configurations`_ for more info on the types of configurations that can be made in this section.
+
+* ``flows``: This section defines the flows that are available to run in your project. See `flow configurations`_ for more info on the types of configurations that can be made in this section.
+
+* ``sources``: This section references tasks and flows from another CumulusCI project on GitHub. See the `use tasks and flows from a different project`_ section for more information.
+
+* ``orgs``: This section defines the scratch org configurations that are available for your project. See `scratch org configurations`_ for more information on the types of configurations that can be made in this section.
+
+* ``plans``: This section contains any custom plans purposely defined to install your project into a customer org. For more information, see `configuring plans in MetaDeploy<TODO>`_.
+
+A complete list of values that can be used in each section can be found in `cumulusci.yml reference<TODO>`_.
+
+
+
+Task Configurations
+-------------------
+
+Task configurations are defined under the ``tasks`` section of your ``cumulusci.yml`` file. Each task configuration defines a task which can be run using the ``cci task run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
+
+
+Override a Task Option
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you repeatedly  specify the same value for an option while running a task, you can configure CumulusCI to use that particular value as a default value.
+
+     Example: Let's enforce a 90% code coverage requirement for Apex code in your project. The `run_tests<TODO>`_ task, which executes all Apex Tests in a target org, can enforce code coverage at a given percentage by passing the ``--required_org_code_coverage_percent`` option.
```

Comment:
```suggestion
     Example: Let's enforce a 90% code coverage requirement for Apex code in your project. The `run_tests<TODO>`_ task, which executes all Apex tests in a target org, can enforce code coverage at a given percentage by passing the ``--required_org_code_coverage_percent`` option.
```
Be on the lookout for non-standard capitalization with standard Salesforce terminology. For something like "Apex tests," I'd do a quick Google search for mentions in definitive sources like Salesforce developer guides to suss out what our standard is (this kind of thing isn't explicitly listed in the style guide). If in doubt, just ask.